Source/Core/VideoCommon/VideoBackendBase.cpp

```diff
@@ -227,6 +230,7 @@ const std::vector<std::unique_ptr<VideoBackendBase>>& VideoBackendBase::GetAvail
 #ifdef __APPLE__
     // Emplace the Vulkan backend at the beginning so it takes precedence over OpenGL.
     backends.emplace(backends.begin(), std::make_unique<Vulkan::VideoBackend>());
+    backends.emplace(backends.begin(), std::make_unique<Metal::VideoBackend>());
```

Comment:
Perhaps we should leave Vulkan by default for now. Once the backend matures a little more, we can make it default.